--- gnome-utils-1.4.0.1/configure.in.orig	Fri Jun 22 18:57:57 2001
+++ gnome-utils-1.4.0.1/configure.in	Sun Jun 24 21:40:56 2001
@@ -340,10 +340,6 @@
 AM_CONDITIONAL(HAVE_LIBXML, test "x$GNOME_XML_LIB" != "x")
 AC_SUBST(HAVE_LIBXML)
 
-AC_SUBST(CFLAGS)
-AC_SUBST(CPPFLAGS)
-AC_SUBST(LDFLAGS)
-
 dnl guname
 AC_ARG_ENABLE(guname-capplet,
 [  --disable-guname-capplet  Use guname-capplet to build guname with control center applet support[default=yes]], enable_guname_capplet="$enableval", enable_guname_capplet=yes)
