--- gnome-utils-1.4.0.1/configure.in.orig	Sun Jun 24 21:42:11 2001
+++ gnome-utils-1.4.0.1/configure.in	Sun Jun 24 21:42:36 2001
@@ -173,7 +173,7 @@
 dnl Internationalization
 dnl ******************************* 
 ALL_LINGUAS="az ca cs da de el en_GB es et fi fr ga gl hu it ja ko lt lv nl no pl pt pt_BR ro ru sk sl sv tr uk wa zh_TW.Big5 zh_CN.GB2312"
-AM_GNOME_GETTEXT
+AM_GNU_GETTEXT
 
 dnl *******************************
 dnl Guile checks
