diff -Nru gnome-utils-1.0.51/cromagnon/Makefile.am gnome-utils-1.0.51.new/cromagnon/Makefile.am
--- gnome-utils-1.0.51/cromagnon/Makefile.am	Sun Mar 12 00:57:37 2000
+++ gnome-utils-1.0.51.new/cromagnon/Makefile.am	Sun Mar 12 00:57:08 2000
@@ -1,4 +1,4 @@
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
 GNOMEG = $(bindir)/gnomeg
 
 bin_SCRIPTS = cromagnon
diff -Nru gnome-utils-1.0.51/gcalc/Makefile.am gnome-utils-1.0.51.new/gcalc/Makefile.am
--- gnome-utils-1.0.51/gcalc/Makefile.am	Sun Mar 12 00:57:37 2000
+++ gnome-utils-1.0.51.new/gcalc/Makefile.am	Sun Mar 12 00:57:08 2000
@@ -1,4 +1,4 @@
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
 
 INCLUDES = 						\
 	-DGNOMELOCALEDIR=\""$(datadir)/locale"\" 	\
diff -Nru gnome-utils-1.0.51/gdiskfree/Makefile.am gnome-utils-1.0.51.new/gdiskfree/Makefile.am
--- gnome-utils-1.0.51/gdiskfree/Makefile.am	Sun Mar 12 00:57:37 2000
+++ gnome-utils-1.0.51.new/gdiskfree/Makefile.am	Sun Mar 12 00:57:08 2000
@@ -21,6 +21,6 @@
 
 EXTRA_DIST = gdiskfree.desktop
 
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
 Utilities_DATA = gdiskfree.desktop
 
diff -Nru gnome-utils-1.0.51/gsearchtool/Makefile.am gnome-utils-1.0.51.new/gsearchtool/Makefile.am
--- gnome-utils-1.0.51/gsearchtool/Makefile.am	Sun Mar 12 00:57:37 2000
+++ gnome-utils-1.0.51.new/gsearchtool/Makefile.am	Sun Mar 12 00:57:08 2000
@@ -1,4 +1,4 @@
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
 
 SUBDIRS = help
 
diff -Nru gnome-utils-1.0.51/gstripchart/Makefile.am gnome-utils-1.0.51.new/gstripchart/Makefile.am
--- gnome-utils-1.0.51/gstripchart/Makefile.am	Sun Mar 12 00:57:08 2000
+++ gnome-utils-1.0.51.new/gstripchart/Makefile.am	Sun Mar 12 00:58:23 2000
@@ -25,7 +25,7 @@
 confdir = $(datadir)/gstripchart
 conf_DATA = gstripchart.conf
 
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
 Utilities_DATA = gstripchart.desktop
 
 if BUILD_GSTRIPCHART_APPLET
diff -Nru gnome-utils-1.0.51/gtt/Makefile.am gnome-utils-1.0.51.new/gtt/Makefile.am
--- gnome-utils-1.0.51/gtt/Makefile.am	Sun Mar 12 00:57:37 2000
+++ gnome-utils-1.0.51.new/gtt/Makefile.am	Sun Mar 12 00:57:08 2000
@@ -36,7 +36,7 @@
 EXTRA_DIST = \
 	gtt.desktop
 
-Productivitydir = $(datadir)/gnome/apps/Applications
+Productivitydir = $(datadir)/applnk/Applications
 Productivity_DATA = gtt.desktop
 
 
diff -Nru gnome-utils-1.0.51/mini-utils/gcolorsel/Makefile.am gnome-utils-1.0.51.new/mini-utils/gcolorsel/Makefile.am
--- gnome-utils-1.0.51/mini-utils/gcolorsel/Makefile.am	Sun Mar 12 00:57:37 2000
+++ gnome-utils-1.0.51.new/mini-utils/gcolorsel/Makefile.am	Sun Mar 12 00:57:08 2000
@@ -1,4 +1,4 @@
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
 
 INCLUDES = 						\
 	-DGNOMELOCALEDIR=\""$(datadir)/locale"\" 	\
diff -Nru gnome-utils-1.0.51/mini-utils/gfontsel/Makefile.am gnome-utils-1.0.51.new/mini-utils/gfontsel/Makefile.am
--- gnome-utils-1.0.51/mini-utils/gfontsel/Makefile.am	Sun Feb  6 16:54:16 2000
+++ gnome-utils-1.0.51.new/mini-utils/gfontsel/Makefile.am	Sun Mar 12 00:58:01 2000
@@ -1,6 +1,6 @@
 SUBDIRS = help
 
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
 
 INCLUDES = 						\
 	-DGNOMELOCALEDIR=\""$(datadir)/locale"\" 	\
diff -Nru gnome-utils-1.0.51/mini-utils/gless/Makefile.am gnome-utils-1.0.51.new/mini-utils/gless/Makefile.am
--- gnome-utils-1.0.51/mini-utils/gless/Makefile.am	Sun Mar 12 00:57:37 2000
+++ gnome-utils-1.0.51.new/mini-utils/gless/Makefile.am	Sun Mar 12 00:57:08 2000
@@ -15,7 +15,7 @@
 	$(GNOMEUI_LIBS) 	\
 	$(INTLLIBS)
 
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
 Utilities_DATA = gless.desktop
 
 EXTRA_DIST = gless.desktop
diff -Nru gnome-utils-1.0.51/mini-utils/gnome-run/Makefile.am gnome-utils-1.0.51.new/mini-utils/gnome-run/Makefile.am
--- gnome-utils-1.0.51/mini-utils/gnome-run/Makefile.am	Sun Mar 12 00:57:37 2000
+++ gnome-utils-1.0.51.new/mini-utils/gnome-run/Makefile.am	Sun Mar 12 00:57:08 2000
@@ -15,7 +15,7 @@
 	$(GNOMEUI_LIBS) 	\
 	$(INTLLIBS)
 
-Rootdir = $(datadir)/gnome/apps
+Rootdir = $(datadir)/applnk
 Root_DATA = gnome-run.desktop
 
 EXTRA_DIST = gnome-run.desktop
diff -Nru gnome-utils-1.0.51/mini-utils/grun/Makefile.am gnome-utils-1.0.51.new/mini-utils/grun/Makefile.am
--- gnome-utils-1.0.51/mini-utils/grun/Makefile.am	Sun Mar 12 00:57:37 2000
+++ gnome-utils-1.0.51.new/mini-utils/grun/Makefile.am	Sun Mar 12 00:57:08 2000
@@ -15,4 +15,4 @@
 	$(INTLLIBS)
 
 
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
diff -Nru gnome-utils-1.0.51/mini-utils/gshutdown/Makefile.am gnome-utils-1.0.51.new/mini-utils/gshutdown/Makefile.am
--- gnome-utils-1.0.51/mini-utils/gshutdown/Makefile.am	Sun Mar 12 00:57:37 2000
+++ gnome-utils-1.0.51.new/mini-utils/gshutdown/Makefile.am	Sun Mar 12 00:57:08 2000
@@ -23,5 +23,5 @@
 pixmapsdir = $(datadir)/pixmaps
 pixmaps_DATA = gnome-shutdown.png
 
-Systemdir = $(datadir)/gnome/apps/System
+Systemdir = $(datadir)/applnk/System
 System_DATA = gshutdown.desktop
diff -Nru gnome-utils-1.0.51/mini-utils/guname/Makefile.am gnome-utils-1.0.51.new/mini-utils/guname/Makefile.am
--- gnome-utils-1.0.51/mini-utils/guname/Makefile.am	Sun Mar 12 00:57:37 2000
+++ gnome-utils-1.0.51.new/mini-utils/guname/Makefile.am	Sun Mar 12 00:57:08 2000
@@ -36,7 +36,7 @@
 	$(LIBGTOP_EXTRA_LIBS)	\
 	$(GUNAME_CAPPLET_LIBS)
 
-Rootdir = $(datadir)/gnome/apps/Utilities
+Rootdir = $(datadir)/applnk/Utilities
 Root_DATA = guname.desktop
 
 if GUNAME_CAPPLET
diff -Nru gnome-utils-1.0.51/mini-utils/gw/Makefile.am gnome-utils-1.0.51.new/mini-utils/gw/Makefile.am
--- gnome-utils-1.0.51/mini-utils/gw/Makefile.am	Sun Mar 12 00:57:37 2000
+++ gnome-utils-1.0.51.new/mini-utils/gw/Makefile.am	Sun Mar 12 00:57:08 2000
@@ -14,7 +14,7 @@
 	$(GNOMEUI_LIBS) 	\
 	$(INTLLIBS)
 
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
 Utilities_DATA = gw.desktop
 
 pixmapdir = $(datadir)/pixmaps
