diff -urN gnome-utils-2.6.1.orig/configure.in gnome-utils-2.6.1/configure.in
--- gnome-utils-2.6.1.orig/configure.in	2004-04-20 15:24:19.267458712 +0200
+++ gnome-utils-2.6.1/configure.in	2004-04-20 15:26:52.909101632 +0200
@@ -297,7 +297,7 @@
 dnl Internationalization
 dnl ******************************* 
 
-ALL_LINGUAS="am ar az be bg bn ca cs cy da de el en_CA en_GB es et eu fa fi fr ga gl gu he hi hr hu id it ja ko lt lv mk ml mn ms ne nl nn no pa pl pt pt_BR ro ru sk sl sq sr sr@Latn sv ta th tr uk vi wa zh_CN zh_TW"
+ALL_LINGUAS="am ar az be bg bn ca cs cy da de el en_CA en_GB es et eu fa fi fr ga gl gu he hi hr hu id it ja ko lt lv mk ml mn ms nb ne nl nn pa pl pt pt_BR ro ru sk sl sq sr sr@Latn sv ta th tr uk vi wa zh_CN zh_TW"
 GETTEXT_PACKAGE=gnome-utils-2.0
 AC_SUBST(GETTEXT_PACKAGE)
 AM_GLIB_GNU_GETTEXT
