diff -aurN gnome-utils-2.13.91.orig/gfloppy/gfloppy.desktop.in gnome-utils-2.13.91/gfloppy/gfloppy.desktop.in
--- gnome-utils-2.13.91.orig/gfloppy/gfloppy.desktop.in	2006-01-22 01:06:50.000000000 +0100
+++ gnome-utils-2.13.91/gfloppy/gfloppy.desktop.in	2006-01-30 18:45:04.290754000 +0100
@@ -7,7 +7,7 @@
 Icon=gnome-dev-floppy
 Type=Application
 StartupNotify=true
-Categories=GTK;GNOME;Application;System;
+Categories=GTK;GNOME;System;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-utils
 X-GNOME-Bugzilla-Component=gfloppy
diff -aurN gnome-utils-2.13.91.orig/gnome-dictionary/data/gnome-dictionary.desktop.in.in gnome-utils-2.13.91/gnome-dictionary/data/gnome-dictionary.desktop.in.in
--- gnome-utils-2.13.91.orig/gnome-dictionary/data/gnome-dictionary.desktop.in.in	2006-01-22 01:06:44.000000000 +0100
+++ gnome-utils-2.13.91/gnome-dictionary/data/gnome-dictionary.desktop.in.in	2006-01-30 18:45:04.290754000 +0100
@@ -7,7 +7,7 @@
 Type=Application
 Icon=gnome-dictionary.png
 StartupNotify=true
-Categories=GNOME;GTK;Application;Utility;
+Categories=GTK;GNOME;Utility;
 X-GNOME-DocPath=gnome-dictionary/gnome-dictionary.xml
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-utils
diff -aurN gnome-utils-2.13.91.orig/gnome-screenshot/gnome-screenshot.desktop.in gnome-utils-2.13.91/gnome-screenshot/gnome-screenshot.desktop.in
--- gnome-utils-2.13.91.orig/gnome-screenshot/gnome-screenshot.desktop.in	2006-01-22 01:06:50.000000000 +0100
+++ gnome-utils-2.13.91/gnome-screenshot/gnome-screenshot.desktop.in	2006-01-30 18:45:19.420754000 +0100
@@ -7,7 +7,7 @@
 Type=Application
 Icon=applets-screenshooter
 StartupNotify=true
-Categories=GTK;GNOME;Application;Utility;
+Categories=GTK;GNOME;Utility;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-utils
 X-GNOME-Bugzilla-Component=screenshot
diff -aurN gnome-utils-2.13.91.orig/gsearchtool/gnome-search-tool.desktop.in gnome-utils-2.13.91/gsearchtool/gnome-search-tool.desktop.in
--- gnome-utils-2.13.91.orig/gsearchtool/gnome-search-tool.desktop.in	2006-01-21 21:18:47.000000000 +0100
+++ gnome-utils-2.13.91/gsearchtool/gnome-search-tool.desktop.in	2006-01-30 18:45:04.290754000 +0100
@@ -7,7 +7,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;GTK;Application;Core;
+Categories=GTK;GNOME;Utility;
 X-GNOME-DocPath=gnome-search-tool/gnome-search-tool.xml
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-utils
diff -aurN gnome-utils-2.13.91.orig/logview/gnome-system-log.desktop.in.in gnome-utils-2.13.91/logview/gnome-system-log.desktop.in.in
--- gnome-utils-2.13.91.orig/logview/gnome-system-log.desktop.in.in	2006-01-25 17:28:53.000000000 +0100
+++ gnome-utils-2.13.91/logview/gnome-system-log.desktop.in.in	2006-01-30 18:45:04.290754000 +0100
@@ -7,7 +7,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GTK;GNOME;Application;System;Monitor;X-Red-Hat-Base;
+Categories=GTK;GNOME;System;
 X-GNOME-DocPath=gnome-system-log/gnome-system-log.xml
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-utils
