--- gnome-utils-1.4.0.2/configure.in.orig	Mon Jul 23 06:51:46 2001
+++ gnome-utils-1.4.0.2/configure.in	Mon Jul 23 06:52:35 2001
@@ -173,7 +173,7 @@
 dnl Internationalization
 dnl ******************************* 
 ALL_LINGUAS="az ca cs da de el en_GB es et fi fr ga gl hu it ja ko lt lv nl no pl pt pt_BR ro ru sk sl sv ta tr uk wa zh_TW.Big5 zh_CN.GB2312"
-AM_GNOME_GETTEXT
+AM_GNU_GETTEXT
 
 dnl *******************************
 dnl Guile checks
