diff -aurN gnome-utils-2.15.3.orig/gnome-dictionary/libgdict/Makefile.am gnome-utils-2.15.3/gnome-dictionary/libgdict/Makefile.am
--- gnome-utils-2.15.3.orig/gnome-dictionary/libgdict/Makefile.am	2006-06-11 14:53:07.000000000 +0200
+++ gnome-utils-2.15.3/gnome-dictionary/libgdict/Makefile.am	2006-06-13 19:01:31.297411472 +0200
@@ -112,8 +112,6 @@
 	$(top_srcdir)/gnome-dictionary/libgdict/gdict-client-context.h \
 	$(top_srcdir)/gnome-dictionary/libgdict/gdict-defbox.h \
 	$(top_srcdir)/gnome-dictionary/libgdict/gdict-entry.h \
-	$(top_srcdir)/gnome-dictionary/libgdict/gdict-database-chooser.h \
-	$(top_srcdir)/gnome-dictionary/libgdict/gdict-strategy-chooser.h \
 	$(top_srcdir)/gnome-dictionary/libgdict/gdict-source.h \
 	$(top_srcdir)/gnome-dictionary/libgdict/gdict-source-loader.h \
 	$(top_srcdir)/gnome-dictionary/libgdict/gdict-speller.h \
