diff -Nru gnome-utils-2.5.1.orig/gfloppy/src/gfloppy.c gnome-utils-2.5.1/gfloppy/src/gfloppy.c
--- gnome-utils-2.5.1.orig/gfloppy/src/gfloppy.c	2003-05-31 08:19:31.000000000 +0200
+++ gnome-utils-2.5.1/gfloppy/src/gfloppy.c	2003-12-31 12:48:30.015351536 +0100
@@ -32,6 +32,7 @@
 #include <sys/ioctl.h>
 #include <sys/wait.h>
 #include <linux/major.h>
+#include <linux/kdev_t.h>
 #include <string.h>
 
 #ifdef HAVE_LINUX_FD_H
