--- gnome-utils-1.0.1/mini-utils/guname/guname.c.sparkle	Thu Mar 25 15:49:32 1999
+++ gnome-utils-1.0.1/mini-utils/guname/guname.c	Thu Mar 25 15:50:04 1999
@@ -358,7 +358,10 @@
   GdkFont * font;
   gchar * s;
   GtkWidget * marquee_frame, * align;
-
+#ifdef WITH_SPARKLES
+  GdkImlibImage *im;
+  GtkWidget     *image;
+#endif
   hbox = gtk_hbox_new(FALSE,GNOME_PAD_SMALL);
   buttonbox = gtk_vbox_new(FALSE,GNOME_PAD_SMALL);
   gtk_container_border_width(GTK_CONTAINER(buttonbox), GNOME_PAD);
