--- gnome-utils-1.1.0/gfloppy/Makefile.am.wiget	Sun Apr 23 07:55:02 2000
+++ gnome-utils-1.1.0/gfloppy/Makefile.am	Wed Apr 26 08:51:06 2000
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-appsdir = $(datadir)/gnome/apps
+appsdir = $(datadir)/applnk
 Utilitiesdir = $(appsdir)/Utilities
 Mimedir = $(datadir)/mime-info
 
--- gnome-utils-1.1.0/gcalc/Makefile.am.wiget	Thu Feb 18 00:27:29 1999
+++ gnome-utils-1.1.0/gcalc/Makefile.am	Wed Apr 26 08:50:17 2000
@@ -1,4 +1,4 @@
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
 
 INCLUDES = 						\
 	-DGNOMELOCALEDIR=\""$(datadir)/locale"\" 	\
--- gnome-utils-1.1.0/gsearchtool/Makefile.am.wiget	Mon Feb  7 23:54:05 2000
+++ gnome-utils-1.1.0/gsearchtool/Makefile.am	Wed Apr 26 08:50:17 2000
@@ -1,4 +1,4 @@
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
 
 SUBDIRS = help
 
--- gnome-utils-1.1.0/gdiskfree/Makefile.am.wiget	Sun Apr 16 21:29:00 2000
+++ gnome-utils-1.1.0/gdiskfree/Makefile.am	Wed Apr 26 08:50:42 2000
@@ -21,6 +21,6 @@
 
 EXTRA_DIST = gdiskfree.desktop
 
-Systemdir = $(datadir)/gnome/apps/System
+Systemdir = $(datadir)/applnk/System
 System_DATA = gdiskfree.desktop
 
--- gnome-utils-1.1.0/gstripchart/Makefile.am.wiget	Sat Apr 22 20:46:35 2000
+++ gnome-utils-1.1.0/gstripchart/Makefile.am	Wed Apr 26 08:50:17 2000
@@ -25,7 +25,7 @@
 confdir = $(datadir)/gstripchart
 conf_DATA = gstripchart.conf
 
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
 Utilities_DATA = gstripchart.desktop
 
 if BUILD_GSTRIPCHART_APPLET
--- gnome-utils-1.1.0/gtt/Makefile.am.wiget	Thu Apr 20 00:43:49 2000
+++ gnome-utils-1.1.0/gtt/Makefile.am	Wed Apr 26 08:50:17 2000
@@ -38,7 +38,7 @@
 EXTRA_DIST = \
 	gtt.desktop
 
-Productivitydir = $(datadir)/gnome/apps/Applications
+Productivitydir = $(datadir)/applnk/Applications
 Productivity_DATA = gtt.desktop
 
 
--- gnome-utils-1.1.0/gdictsrc/Makefile.am.wiget	Sat Apr  8 17:34:56 2000
+++ gnome-utils-1.1.0/gdictsrc/Makefile.am	Wed Apr 26 08:55:42 2000
@@ -34,5 +34,5 @@
 gdictappletdir  = $(datadir)/applets/Utility
 gdictapplet_DATA = gdict.desktop
 
-gdictappdir  = $(datadir)/gnome/apps/Utilities
+gdictappdir  = $(datadir)/applnk/Utilities
 gdictapp_DATA = GDict.desktop
--- gnome-utils-1.1.0/cromagnon/Makefile.am.wiget	Thu Feb 18 00:27:24 1999
+++ gnome-utils-1.1.0/cromagnon/Makefile.am	Wed Apr 26 08:50:17 2000
@@ -1,4 +1,4 @@
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
 GNOMEG = $(bindir)/gnomeg
 
 bin_SCRIPTS = cromagnon
--- gnome-utils-1.1.0/mini-utils/gshutdown/Makefile.am.wiget	Thu Apr 20 03:59:48 2000
+++ gnome-utils-1.1.0/mini-utils/gshutdown/Makefile.am	Wed Apr 26 08:50:20 2000
@@ -23,5 +23,5 @@
 pixmapsdir = $(datadir)/pixmaps
 pixmaps_DATA = gnome-shutdown.png
 
-Systemdir = $(datadir)/gnome/apps/System
+Systemdir = $(datadir)/applnk/System
 System_DATA = gshutdown.desktop
--- gnome-utils-1.1.0/mini-utils/guname/Makefile.am.wiget	Sun Apr 16 21:29:01 2000
+++ gnome-utils-1.1.0/mini-utils/guname/Makefile.am	Wed Apr 26 08:53:56 2000
@@ -37,7 +37,7 @@
 	$(LIBGTOP_EXTRA_LIBS)	\
 	$(GUNAME_CAPPLET_LIBS)
 
-Systemdir = $(datadir)/gnome/apps/System
+Systemdir = $(datadir)/applnk/System
 System_DATA = guname.desktop
 
 if GUNAME_CAPPLET
--- gnome-utils-1.1.0/mini-utils/gw/Makefile.am.wiget	Mon Apr 24 04:45:47 2000
+++ gnome-utils-1.1.0/mini-utils/gw/Makefile.am	Wed Apr 26 08:54:18 2000
@@ -17,7 +17,7 @@
 	$(GNOMEUI_LIBS) 	\
 	$(INTLLIBS)
 
-Systemdir = $(datadir)/gnome/apps/System
+Systemdir = $(datadir)/applnk/System
 System_DATA = gw.desktop
 
 pixmapdir = $(datadir)/pixmaps
--- gnome-utils-1.1.0/mini-utils/gless/Makefile.am.wiget	Tue Apr 11 22:43:52 2000
+++ gnome-utils-1.1.0/mini-utils/gless/Makefile.am	Wed Apr 26 08:50:17 2000
@@ -18,7 +18,7 @@
 	$(GNOMEUI_LIBS) 	\
 	$(INTLLIBS)
 
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
 Utilities_DATA = gless.desktop
 
 EXTRA_DIST = gless.desktop
--- gnome-utils-1.1.0/mini-utils/gfontsel/Makefile.am.wiget	Fri Feb  4 03:10:46 2000
+++ gnome-utils-1.1.0/mini-utils/gfontsel/Makefile.am	Wed Apr 26 08:50:17 2000
@@ -1,6 +1,6 @@
 SUBDIRS = help
 
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
 
 INCLUDES = 						\
 	-DGNOMELOCALEDIR=\""$(datadir)/locale"\" 	\
--- gnome-utils-1.1.0/mini-utils/gcolorsel/Makefile.am.wiget	Tue Apr 25 17:32:22 2000
+++ gnome-utils-1.1.0/mini-utils/gcolorsel/Makefile.am	Wed Apr 26 08:53:04 2000
@@ -3,7 +3,7 @@
 SUBDIRS = glade doc
 
 gladedir = $(datadir)/gcolorsel/glade
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
 
 CFLAGS += -DGCOLORSEL_GLADEDIR=\""$(gladedir)/"\" -g
 
