--- gnome-utils-1.2.0/gdictsrc/main.c.errordialog	Mon Aug 21 12:35:00 2000
+++ gnome-utils-1.2.0/gdictsrc/main.c	Mon Aug 21 12:37:27 2000
@@ -88,7 +88,11 @@
     gdict_app_create ();
     gdict_pref_load ();
     if ((gdict_init_context () < 0) && (gdict_applet_toggle == FALSE))
-	    return 0;
+      {
+            GtkWidget *dialog = gnome_error_dialog (_("Couldn't start the dictionary; most likely, http://www.dict.org could not be contacted."));
+            gnome_dialog_run (dialog);
+	    return 1;
+      }
     /* If we can not find the context then we may as well just quit
      * since nothing is usable anymore. But not if we're an applet */
     defbox->context = context;
