--- gnome-utils-1.2.0/mini-utils/guname/guname.c.nomailx	Sun Aug  6 01:08:34 2000
+++ gnome-utils-1.2.0/mini-utils/guname/guname.c	Sun Aug  6 01:08:42 2000
@@ -782,7 +782,7 @@
     
     /* This isn't translated; maybe good, since most tech support email should be
        in English? don't know. */
-    command = g_strconcat("mailx -s \"System Information for host ", 
+    command = g_strconcat("mail -s \"System Information for host ", 
                              info[si_host] ? info[si_host] : "Unknown",
                              "\" ", data->to, NULL);
     
