diff -urN aa/gnome-utils-2.8.0/gfloppy/doc/ja/gfloppy-ja.omf gnome-utils-2.8.0/gfloppy/doc/ja/gfloppy-ja.omf
--- aa/gnome-utils-2.8.0/gfloppy/doc/ja/gfloppy-ja.omf	2003-02-23 04:11:14.000000000 +0100
+++ gnome-utils-2.8.0/gfloppy/doc/ja/gfloppy-ja.omf	2004-09-13 13:32:47.463462288 +0200
@@ -7,9 +7,6 @@
     <title>
       フロッピー初期化ツール マニュアル
     </title>
-    <contributor>
-      emerald@gnome.gr.jp (Japan GNOME Users Group)
-    </contributor>
     <date>
       January 27, 2003
     </date>
