--- gnome-utils-1.4.0/configure.in~	Tue Mar 13 03:16:11 2001
+++ gnome-utils-1.4.0/configure.in	Tue Apr 24 04:54:07 2001
@@ -77,7 +77,7 @@
 dnl Internationalization
 dnl ******************************* 
 ALL_LINGUAS="az ca cs da de el en_GB es et fi fr ga gl hu it ja ko lt nl no pl pt pt_BR ro ru sk sl sv tr uk wa zh_TW.Big5 zh_CN.GB2312"
-AM_GNOME_GETTEXT
+AM_GNU_GETTEXT
 
 dnl *******************************
 dnl Guile checks
