diff -Nru gnome-utils-1.2.0/cromagnon/Makefile.am gnome-utils-1.2.0.new/cromagnon/Makefile.am
--- gnome-utils-1.2.0/cromagnon/Makefile.am	Mon Jun 12 10:56:03 2000
+++ gnome-utils-1.2.0.new/cromagnon/Makefile.am	Mon Jun 12 10:55:28 2000
@@ -1,4 +1,4 @@
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
 GNOMEG = $(bindir)/gnomeg
 
 bin_SCRIPTS = cromagnon
diff -Nru gnome-utils-1.2.0/gcalc/Makefile.am gnome-utils-1.2.0.new/gcalc/Makefile.am
--- gnome-utils-1.2.0/gcalc/Makefile.am	Mon Jun 12 10:56:03 2000
+++ gnome-utils-1.2.0.new/gcalc/Makefile.am	Mon Jun 12 10:55:28 2000
@@ -1,4 +1,4 @@
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
 
 INCLUDES = 						\
 	-DGNOMELOCALEDIR=\""$(datadir)/locale"\" 	\
diff -Nru gnome-utils-1.2.0/gcharmap/Makefile.am gnome-utils-1.2.0.new/gcharmap/Makefile.am
--- gnome-utils-1.2.0/gcharmap/Makefile.am	Fri May 19 20:57:44 2000
+++ gnome-utils-1.2.0.new/gcharmap/Makefile.am	Mon Jun 12 10:56:29 2000
@@ -1,6 +1,6 @@
 # Process this file with automake to produce Makefile.in
 
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
 
 EXTRA_DIST = AUTHORS README TODO BUGS gcharmap.desktop 
 SUBDIRS = src doc pixmaps
diff -Nru gnome-utils-1.2.0/gdictsrc/Makefile.am gnome-utils-1.2.0.new/gdictsrc/Makefile.am
--- gnome-utils-1.2.0/gdictsrc/Makefile.am	Mon Jun 12 10:56:03 2000
+++ gnome-utils-1.2.0.new/gdictsrc/Makefile.am	Mon Jun 12 10:55:28 2000
@@ -36,5 +36,5 @@
 gdictappletdir  = $(datadir)/applets/Utility
 gdictapplet_DATA = gdict.desktop
 
-gdictappdir  = $(datadir)/gnome/apps/Utilities
+gdictappdir  = $(datadir)/applnk/Utilities
 gdictapp_DATA = GDict.desktop
diff -Nru gnome-utils-1.2.0/gdiskfree/Makefile.am gnome-utils-1.2.0.new/gdiskfree/Makefile.am
--- gnome-utils-1.2.0/gdiskfree/Makefile.am	Mon Jun 12 10:56:03 2000
+++ gnome-utils-1.2.0.new/gdiskfree/Makefile.am	Mon Jun 12 10:55:28 2000
@@ -21,6 +21,6 @@
 
 EXTRA_DIST = gdiskfree.desktop
 
-Systemdir = $(datadir)/gnome/apps/System
+Systemdir = $(datadir)/applnk/System
 System_DATA = gdiskfree.desktop
 
diff -Nru gnome-utils-1.2.0/gfloppy/Makefile.am gnome-utils-1.2.0.new/gfloppy/Makefile.am
--- gnome-utils-1.2.0/gfloppy/Makefile.am	Mon Jun 12 10:56:03 2000
+++ gnome-utils-1.2.0.new/gfloppy/Makefile.am	Mon Jun 12 10:55:28 2000
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-appsdir = $(datadir)/gnome/apps
+appsdir = $(datadir)/applnk
 Utilitiesdir = $(appsdir)/Utilities
 Mimedir = $(datadir)/mime-info
 
diff -Nru gnome-utils-1.2.0/gsearchtool/Makefile.am gnome-utils-1.2.0.new/gsearchtool/Makefile.am
--- gnome-utils-1.2.0/gsearchtool/Makefile.am	Mon Jun 12 10:56:03 2000
+++ gnome-utils-1.2.0.new/gsearchtool/Makefile.am	Mon Jun 12 10:55:28 2000
@@ -1,4 +1,4 @@
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
 
 SUBDIRS = help
 
diff -Nru gnome-utils-1.2.0/gstripchart/Makefile.am gnome-utils-1.2.0.new/gstripchart/Makefile.am
--- gnome-utils-1.2.0/gstripchart/Makefile.am	Mon Jun 12 10:56:03 2000
+++ gnome-utils-1.2.0.new/gstripchart/Makefile.am	Mon Jun 12 10:55:28 2000
@@ -25,7 +25,7 @@
 confdir = $(datadir)/gstripchart
 conf_DATA = gstripchart.conf
 
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
 Utilities_DATA = gstripchart.desktop
 
 if BUILD_GSTRIPCHART_APPLET
diff -Nru gnome-utils-1.2.0/gtt/Makefile.am gnome-utils-1.2.0.new/gtt/Makefile.am
--- gnome-utils-1.2.0/gtt/Makefile.am	Mon Jun 12 10:56:03 2000
+++ gnome-utils-1.2.0.new/gtt/Makefile.am	Mon Jun 12 10:55:28 2000
@@ -38,7 +38,7 @@
 EXTRA_DIST = \
 	gtt.desktop
 
-Productivitydir = $(datadir)/gnome/apps/Applications
+Productivitydir = $(datadir)/applnk/Office
 Productivity_DATA = gtt.desktop
 
 
diff -Nru gnome-utils-1.2.0/logview/Makefile.am gnome-utils-1.2.0.new/logview/Makefile.am
--- gnome-utils-1.2.0/logview/Makefile.am	Fri May  5 07:36:31 2000
+++ gnome-utils-1.2.0.new/logview/Makefile.am	Mon Jun 12 10:57:49 2000
@@ -2,7 +2,7 @@
 
 SUBDIRS = pixmaps
 
-Utilitiesdir = $(datadir)/gnome/apps/System
+Utilitiesdir = $(datadir)/applnk/System
 additionaldir  = $(datadir)/logview
 
 INCLUDES = -DG_LOG_DOMAIN=\"logview\"					\
diff -Nru gnome-utils-1.2.0/mini-utils/gcolorsel/Makefile.am gnome-utils-1.2.0.new/mini-utils/gcolorsel/Makefile.am
--- gnome-utils-1.2.0/mini-utils/gcolorsel/Makefile.am	Mon Jun 12 10:56:03 2000
+++ gnome-utils-1.2.0.new/mini-utils/gcolorsel/Makefile.am	Mon Jun 12 10:55:28 2000
@@ -3,7 +3,7 @@
 SUBDIRS = glade doc
 
 gladedir = $(datadir)/gcolorsel/glade
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
 
 CFLAGS += -DGCOLORSEL_GLADEDIR=\""$(gladedir)/"\" -g
 
diff -Nru gnome-utils-1.2.0/mini-utils/gfontsel/Makefile.am gnome-utils-1.2.0.new/mini-utils/gfontsel/Makefile.am
--- gnome-utils-1.2.0/mini-utils/gfontsel/Makefile.am	Mon Jun 12 10:56:03 2000
+++ gnome-utils-1.2.0.new/mini-utils/gfontsel/Makefile.am	Mon Jun 12 10:55:28 2000
@@ -1,6 +1,6 @@
 SUBDIRS = help
 
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
 
 INCLUDES = 						\
 	-DGNOMELOCALEDIR=\""$(datadir)/locale"\" 	\
diff -Nru gnome-utils-1.2.0/mini-utils/gless/Makefile.am gnome-utils-1.2.0.new/mini-utils/gless/Makefile.am
--- gnome-utils-1.2.0/mini-utils/gless/Makefile.am	Mon Jun 12 10:56:03 2000
+++ gnome-utils-1.2.0.new/mini-utils/gless/Makefile.am	Mon Jun 12 10:55:28 2000
@@ -18,7 +18,7 @@
 	$(GNOMEUI_LIBS) 	\
 	$(INTLLIBS)
 
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
 Utilities_DATA = gless.desktop
 
 EXTRA_DIST = gless.desktop
diff -Nru gnome-utils-1.2.0/mini-utils/gshutdown/Makefile.am gnome-utils-1.2.0.new/mini-utils/gshutdown/Makefile.am
--- gnome-utils-1.2.0/mini-utils/gshutdown/Makefile.am	Mon Jun 12 10:56:03 2000
+++ gnome-utils-1.2.0.new/mini-utils/gshutdown/Makefile.am	Mon Jun 12 10:55:28 2000
@@ -23,5 +23,5 @@
 pixmapsdir = $(datadir)/pixmaps
 pixmaps_DATA = gnome-shutdown.png
 
-Systemdir = $(datadir)/gnome/apps/System
+Systemdir = $(datadir)/applnk/System
 System_DATA = gshutdown.desktop
diff -Nru gnome-utils-1.2.0/mini-utils/guname/Makefile.am gnome-utils-1.2.0.new/mini-utils/guname/Makefile.am
--- gnome-utils-1.2.0/mini-utils/guname/Makefile.am	Mon Jun 12 10:56:03 2000
+++ gnome-utils-1.2.0.new/mini-utils/guname/Makefile.am	Mon Jun 12 10:55:28 2000
@@ -39,7 +39,7 @@
 	$(LIBGTOP_EXTRA_LIBS)	\
 	$(GUNAME_CAPPLET_LIBS)
 
-Systemdir = $(datadir)/gnome/apps/System
+Systemdir = $(datadir)/applnk/System
 System_DATA = guname.desktop
 
 if GUNAME_CAPPLET
diff -Nru gnome-utils-1.2.0/mini-utils/gw/Makefile.am gnome-utils-1.2.0.new/mini-utils/gw/Makefile.am
--- gnome-utils-1.2.0/mini-utils/gw/Makefile.am	Mon Jun 12 10:56:03 2000
+++ gnome-utils-1.2.0.new/mini-utils/gw/Makefile.am	Mon Jun 12 10:55:28 2000
@@ -17,7 +17,7 @@
 	$(GNOMEUI_LIBS) 	\
 	$(INTLLIBS)
 
-Systemdir = $(datadir)/gnome/apps/System
+Systemdir = $(datadir)/applnk/System
 System_DATA = gw.desktop
 
 pixmapdir = $(datadir)/pixmaps
diff -Nru gnome-utils-1.2.0/mini-utils/idetool/Makefile.am gnome-utils-1.2.0.new/mini-utils/idetool/Makefile.am
--- gnome-utils-1.2.0/mini-utils/idetool/Makefile.am	Thu May 11 03:13:09 2000
+++ gnome-utils-1.2.0.new/mini-utils/idetool/Makefile.am	Mon Jun 12 10:58:41 2000
@@ -24,5 +24,5 @@
 
 EXTRA_DIST  = $(BITMAPS) idetool.desktop
 
-Utilitiesdir = $(datadir)/gnome/apps/Utilities
+Utilitiesdir = $(datadir)/applnk/Utilities
 Utilities_DATA = idetool.desktop
