--- gnome-utils-2.14.0/configure.in~	2007-02-03 23:46:22.000000000 +0100
+++ gnome-utils-2.14.0/configure.in	2007-02-03 23:46:22.000000000 +0100
@@ -412,10 +412,7 @@
 dnl Internationalization
 dnl ******************************* 
 
-ALL_LINGUAS="am ar az be bg bn bs ca cs cy da de el en_CA en_GB es et eu
-fa fi fr ga gl gu he hi hr hu id it ja ko ku lt lv mk ml mn ms nb ne nl
-nn no pa pl pt pt_BR ro ru rw sk sl sq sr sr@Latn sv ta th tr uk vi wa
-xh zh_CN zh_HK zh_TW"
+ALL_LINGUAS="am ar az be bg bn bs ca cs cy da de el en_CA en_GB es et eu fa fi fr ga gl gu he hi hr hu id it ja ko ku lt lv mk ml mn ms nb ne nl nn no pa pl pt pt_BR ro ru rw sk sl sq sr sr@Latn sv ta th tr uk vi wa xh zh_CN zh_HK zh_TW"
 
 GETTEXT_PACKAGE=gnome-utils-2.0
 AC_DEFINE_UNQUOTED([GETTEXT_PACKAGE], "$GETTEXT_PACKAGE", [Define the gettext
